Uchu.World/Handlers/GameMessages/GeneralHandler.cs

```diff
@@ -74,6 +74,7 @@ public async Task ReadyForUpdatesHandler(ReadyForUpdateMessage message, Player p
         {
             Logger.Debug($"Loaded: {message.GameObject}");
             await player.OnReadyForUpdatesEvent.InvokeAsync(message);
+            Zone.SendSerialization(message.GameObject,new []{ player });
```

Comment:
Here I am again, being really nit picky regarding code style :P

`message.GameObject, new []{ player }`

Reply:
I was wondering if I was going to get away with this after seeing this in the previous PR (which was after I made this commit).